src/components/App.js

```diff
@@ -43,7 +43,7 @@ const App = () => {
   }, []);
 
   return (
-    <>
+    <Router>
```

Comment:
we don't need the Router on the App.js file because we handle it in our boostrapping file. in the src/index.js file we wrap the entire App component inside of a Router. we can remove the import as well for BrowserRouter